docs/source/en/using-diffusers/custom_pipeline_overview.md

```diff
@@ -78,35 +76,35 @@ tokenizer = T5Tokenizer.from_pretrained(pipe_id, subfolder="tokenizer")
 text_encoder = T5EncoderModel.from_pretrained(pipe_id, subfolder="text_encoder")
 ```
 
-2. **Scheduler**
+2. Load a scheduler:
 
 ```python
 from diffusers import DPMSolverMultistepScheduler
 
 scheduler = DPMSolverMultistepScheduler.from_pretrained(pipe_id, subfolder="scheduler")
 ```
 
-3. **Image processor**
+3. Load an image processor:
 
 ```python
 from transformers import CLIPFeatureExtractor
 
 feature_extractor = CLIPFeatureExtractor.from_pretrained(pipe_id, subfolder="feature_extractor")
 ```
 
-Now, you need to implement the custom UNet. The implementation is available [here](https://github.com/showlab/Show-1/blob/main/showone/models/unet_3d_condition.py). So, let's create a Python script called `showone_unet_3d_condition.py` and copy over the implementation, changing the `UNet3DConditionModel` classname to `ShowOneUNet3DConditionModel` to avoid any conflicts with Diffusers. This is because Diffusers already has one `UNet3DConditionModel`. We put all the components needed to implement the class in `showone_unet_3d_condition.py` only. You can find the entire file [here](https://huggingface.co/sayakpaul/show-1-base-with-code/blob/main/unet/showone_unet_3d_condition.py).
+4. Now you'll load a [custom UNet](https://github.com/showlab/Show-1/blob/main/showone/models/unet_3d_condition.py), which in this example, has already been implemented in the `showone_unet_3d_condition.py` [script](https://huggingface.co/sayakpaul/show-1-base-with-code/blob/main/unet/showone_unet_3d_condition.py) for your convenience. You'll notice the `UNet3DConditionModel` class name is changed to `ShowOneUNet3DConditionModel` because [`UNet3DConditionModel`] already exists in Diffusers. Any components needed for the `ShowOneUNet3DConditionModel` class should be placed in the `showone_unet_3d_condition.py` script.
```

Comment:
Can we make it explicitly clear that users will have to do the implementation following this format?